app/assets/javascripts/surveys.js

```diff
@@ -175,6 +175,111 @@
           title: 'Tell us about your visit to GOV.UK',
           surveyCta: 'Answer 2 quick questions to help us make GOV.UK better',
         }
+      },
+      {
+        identifier: 'scrutinise_government',
+        surveyType: 'url',
+        frequency: 6,
+        startTime: new Date('August 25, 2017').getTime(),
+        endTime: new Date('October 1, 2017 23:59:50').getTime(),
+        url: 'https://signup.take-part-in-research.service.gov.uk/?utm_campaign=Scrutinise_government&utm_source=Hold_gov_to_account&utm_medium=gov.uk&t=GDS&id=15',
+        templateArgs: {
+          title: 'Help improve GOV.UK',
+          surveyCta: 'Answer some questions about yourself to join the research community.',
+          surveyCtaPostscript: 'This link opens in a new tab.'
+        },
+        activeWhen: {
+          path: [
+            '^/government/policies(?:/|$)',
+            '^/government/announcements(?:/|$)',
+            '^/government/publications(?:/|$)',
+            '^/government/statistics(?:/|$)',
+            '^/government/publications?publication_filter_options=consultations(?:/|$)',
```

Comment:
We can drop this URL.  Primarily because the current implementation of the surveys code only looks at the `/path` not the `?query=string` of the current page, so this would never match anything, but also because (luckily for us) we already have `/government/publications` above which would cover it.